test/unit/codeshift.test.js

```diff
@@ -10,18 +11,14 @@ describe('Installing codeshift libraries', () => {
     await codeshift.js?.install();
     await codeshift.ruby?.install();
 
-    run = mockCommands({
+    run = await mockCommands({
       npm: () => ({ status: 0 }),
       gem: () => ({ status: 0 }),
       [codeshift.js.bin]: () => ({ status: 0 }),
       [codeshift.ruby.bin]: () => ({ status: 0 })
     });
 
-    mockRequire('fs', {
-      existsSync: (path) => false
-    });
-
-    codeshift = mockRequire.reRequire('../../src/utils').codeshift;
+    spyOn(fs, 'existsSync').and.callFake(p => false);
```

Comment:
I don't think these unit tests mock the filesystem like other tests, so these spies would remain if you were going to search and replace to keep any existing fs spies intact.

Reply:
I believe it is -- I have `setupTest` in `./helpers/index.js` in a `beforeEach` so it's run before any test

Reply:
Gotcha. How do you feel about removing global hooks? This was part of my work recently with the CLI repo and was actually the reason for `setupTest` to come into existence – to remove global hooks. With our suites, global hooks tended to be the source of some harder to track issues. Especially when you try to isolate new tests and some side effects happen just from importing a file.